css/layout.css

```diff
@@ -17,18 +9,17 @@ header.navigation {
   align-items: center;
 }
 
-section.search-container {
+div.feature-components {
```

Comment:
Generally, when you select a class, you don't need to include its tag name. It makes it easier down-the-road if you need to change HTML semantics.